examples/JMX/README.md

```diff
@@ -0,0 +1,10 @@
+# JMX Docker Compose example
+
+In general, all Kafka services uses the `*JMX_OPTS` environment variable to pass JMX options to the JVM. This compose example has been configured to allow external JMX clients (e.g. jconsole) to connect to the containers' JMX port. 
+
+If you log into the container you will see this environment variable being set. This can cause issues if you are running kafka CLI commands locally within the container. To work around this you would need to unset the `*JMX_OPTS` environment variable. For instance for the kafka container, you need to unset the `KAFKA_JMX_OPTS` before you can run any kafka CLI command:
```

Comment:
suggest: `If you are running Kafka CLI commands locally within the container, you must unset the `*JMX_OPTS` environment variable.`